### DISCUSSION TITLE
sound does not match the time

### DISCUSSION THREAD

**Discussion by sergey-boreysha** [MusicPlayerDaemon/MPD#1919] [Q&A]
I generated an mp3 file with beats every fourth second.
When playing, the sound does not match the time.
What is the problam may be.

https://drive.google.com/file/d/1vVhuPIWcK4aieZiw7NpsLP9jpeHIA-jp/view?usp=sharing

**Comment by MaxKellermann** [comment]
Why post this twice? https://github.com/MusicPlayerDaemon/MPD/issues/1912
And why did you refuse to provide the information that was asked from you?

**Reply by sergey-boreysha** [reply]
Sorry, I didn't see what I need to provide?

**Reply by MaxKellermann** [reply]
The information that was asked from you when you opened the issue. Before you wrote your text, you deleted all the instructions on what to provide. This was ignorant of you.

**Reply by sergey-boreysha** [reply]
Sorry, I was in a hurry, I'll correct it...

**Reply by sergey-boreysha** [reply]
Hello Max!
English is not my native language, I find it difficult to communicate
my reports are not fake, I just don’t understand what information I should provide you with...
there are translation features and misunderstandings
need a log during playback, ok, I'll add it
can you be patient and help me?
